aiohttp/web.py

```diff
@@ -221,6 +221,22 @@ def middlewares(self):
         return self._middlewares
 
     def make_handler(self, **kwargs):
+        try:
+            debug = kwargs['debug']
```

Comment:
In the issue description you said:

> If app.debug conflicts with explicitly passed debug param -- raise ValueError.

So that if we put default boolean value to `kwargs.pop()` we can't know if the key was in the dictionary or not.
Also we want to compare this value to `self.debug` and raise a `ValueError` if they differ but only if the value was there.
Of course we can use `None` as a default value and then check for `None`.
```
debug = kwargs.pop('debug', None)
if debug is not None and debug != self.debug:
    raise ValueError("...")
debug = self.debug
return self._handler_factory(self, self.router, debug=debug, loop=self.loop, **kwargs)
```